### ISSUE TITLE
Data Normalization is before splitting but should be after splitting

### ISSUE THREAD

**Issue by abimael-dominguez** [Mortiniera/Data_Science_Portfolio#1]
Hello.

I find your code very useful but I guess I found a mistake.
Data Normalization should be after splitting, is not it?
If you do that before, then the test set will know information of the whole dataset (something that we would like to hide) and the evaluation of the model would be very optimistic.

I found some more information here:

https://datascience.stackexchange.com/questions/54908/data-normalization-before-or-after-train-test-split

"Normalization across instances should be done after splitting the data between training and test set, using only the data from the training set".
